js/console_test.ts

```diff
@@ -76,6 +78,8 @@ test(function consoleTestStringifyCircular() {
   assertEqual(stringify(undefined), "undefined");
   assertEqual(stringify(new Extended()), "Extended { a: 1, b: 2 }");
   assertEqual(stringify(function f() {}), "[Function: f]");
+  assertEqual(stringify(async function af() {}), "[AsyncFunction: af]");
+  assertEqual(stringify(function* gf() {}), "[GeneratorFunction: gf]");
```

Comment:
Please add a test for async generators: `stringify(async function* agf() {})` which returns `[AsyncGeneratorFunction: agf]".